pkg/operator/ceph/config/config.go

```diff
@@ -113,14 +113,37 @@ func SetDefaultConfigs(
 		return errors.Wrapf(err, "failed to apply default Ceph configurations")
 	}
 
+	// When enabled the collector will logrotate logs from files
+	if clusterSpec.LogCollector.Enabled {
+		// Override "log file" for existing clusters since it is empty
+		logOptions := []Option{
+			configOverride("global", "log file", "/var/log/ceph/$cluster-$name.log"),
+			configOverride("global", "log to file", "true"),
+		}
+
+		if err := monStore.SetAll(logOptions...); err != nil {
```

Comment:
nit: `monStore.SetAll()` could be outside the if/else statement, no need to duplicate it just for a slightly different error message.

Reply:
Sometimes it's still useful for debugging IMO.